src/components/NavBar/index.tsx

```diff
@@ -0,0 +1,70 @@
+import { Flex, Button, Text, IconButton, Grid, Menu, MenuButton } from "@chakra-ui/react";
+import { FaRegUser } from "react-icons/fa";
+import { FiDownload } from "react-icons/fi";
+import { HiOutlineShoppingBag } from "react-icons/hi";
+import { IoIosArrowBack } from "react-icons/io";
+import { RiArrowGoBackLine, RiArrowGoForwardLine } from "react-icons/ri";
+import { BiStar, BiPencil } from "react-icons/bi";
+
+const NavigationBar = (): JSX.Element => {
+  return (
+    <Grid templateColumns="repeat(3, 1fr)" bg="brand.primary" p={4}>
+      <Flex alignItems="center">
+        <Button leftIcon={<IoIosArrowBack />} pl="0" variant="navbarIconBtn" data-testid="home">
+          Home
+        </Button>
+        <Menu>
+          <MenuButton as={Button} variant="navbarIconBtn" data-testid="file">
+            File
+          </MenuButton>
+        </Menu>
+        <Flex flex="1" justifyContent="center">
+          <IconButton
+            aria-label="Revert edit"
+            icon={<RiArrowGoBackLine />}
+            variant="navbarIconBtn"
+          />
+          <IconButton
+            aria-label="Forward edit"
+            icon={<RiArrowGoForwardLine />}
+            variant="navbarIconBtn"
+            color="#72818B"
```

Comment:
The logic here is when you have actions that require undo or redo, the icon will become #FFFFFF, but at the beginning, the color of both buttons is "#72818B", Of course it's okay to write this now.